inspire_disambiguation/api.py

```diff
@@ -113,10 +129,15 @@ def train_and_save_distance_model(
         training_model_runtime=str(training_model_time - load_data_to_model_time),
         save_model_runtime=str(save_model_time - training_model_time),
         total_runtime=str(save_model_time - start_time),
+        validation_score=str(validation_score)
     )
+    # validation signatures should be passed then to the clustering function if it's needed
+    if validate:
```

Comment:
this is ugly, better return it always, and then you can choose whether to use it. Don't forget to update the docstring with the return value.